requirements/requirements-django.txt

```diff
@@ -0,0 +1 @@
+Django>=1.8,<2.0
```

Comment:
There is no need for this file if you let `tox` install the package's requirements: https://github.com/edx/edx-drf-extensions/blob/master/tox.ini#L9. Frankly, we *should* do that to ensure that `install_requires` is not missing any requirements.